tests/suites/sanity_test.go

```diff
@@ -11,13 +12,14 @@ import (
 
 	// log "github.com/sirupsen/logrus"
 
+	cassandra "github.com/mesosphere/kudo-cassandra-operator/tests/utils/cassandra"
 	k8s "github.com/mesosphere/kudo-cassandra-operator/tests/utils/k8s"
 	kubectl "github.com/mesosphere/kudo-cassandra-operator/tests/utils/kubectl"
 	kudo "github.com/mesosphere/kudo-cassandra-operator/tests/utils/kudo"
 )
 
 var (
-	TestName          = "simple-install-uninstall-test"
+	TestName          = "sanity-test"
 	OperatorName      = os.Getenv("OPERATOR_NAME")
 	TestNamespace     = fmt.Sprintf("%s-namespace", TestName)
 	TestInstance      = fmt.Sprintf("%s-instance", OperatorName)
```

Comment:
It is not necessary, it's just a way to make it possible to more easily grep for log outputs that are related to the instance itself. Since grepping for `cassandra` would pretty much output every line, making it possible to grep for `cassandra-instance` is valuable.